plover/oslayer/osxkeyboardcontrol.py

```diff
@@ -431,24 +469,27 @@ def _send_sequence(sequence):
         mods_flags = 0
 
         for keycode, key_down in sequence:
-            if not key_down and keycode in MODIFIER_KEYS_TO_MASKS:
-                mods_flags &= ~MODIFIER_KEYS_TO_MASKS[keycode]
+            if keycode >= NX_KEY_OFFSET:
+                KeyboardEmulation._send_media_key(keycode - NX_KEY_OFFSET, key_down)
+            else:
```

Comment:
How about using a continue here, so the rest of the code does not need to be shifted? Alternatively, change `_send_media_key` to `_get_media_event`, and factorize the call to `CGEventPost`? Is the fact that the event tap location is 0 (`kCGHIDEventTap`?) and not `kCGSessionEventTap` required?